### ISSUE TITLE
Problem panel does not capture all errors

### ISSUE THREAD

**Issue by Jiefei-Wang** [James-Yu/LaTeX-Workshop#4123]
I understand the error message is hard to parse, but I wonder if this can be improved. From my log file, I can see the error is due to a missing file:
```
LaTeX Warning: File `normal blood pressure sample mean1.png' not found on input line 706.

c:/Users/jiefei/OneDrive/UTMB/teaching/SPPH 6396 Multivariable Data Analysis/Lecture 3/lecture/lecture 3.tex:706: Package pdftex.def Error: File `normal blood pressure sample mean1.png' not found: using draft setting.

See the pdftex.def package documentation for explanation.
Type  H <return>  for immediate help.
```
However, when I checked my problem panel, none of them was relevant, just a few notes. I checked my compilation flags and `-file-line-error` has been included
![image](https://github.com/James-Yu/LaTeX-Workshop/assets/13570205/d13558e7-55ee-4e15-bd21-1672a774e0df)

**Comment by James-Yu** [comment]
Please submit a feature request and attach the full log messages.
